icenet.c

```diff
@@ -20,6 +20,11 @@
 #include <linux/netdevice.h>
 #include <linux/etherdevice.h>
 
+/* Can't add new CONFIG parameters in an external module, so define them here */
+#define ICENET_MTU 1500
```

Comment:
I suggest keeping the `CONFIG_` prefix to make it clear that these are intended to be parameters, as well as to 
flexibly enable adding the driver back in-tree if ever needed.

```suggestion
#ifndef CONFIG_ICENET_MTU
#define CONFIG_ICENET_MTU 1500
#endif
```